video2dataset/dataloader/dataloader.py

```diff
@@ -96,21 +96,29 @@ def get_video_dataset(
     if keys_to_remove is None:
         keys_to_remove = []
 
+    if isinstance(urls, str):
+        urls = [urls]
+    use_torchdata = urls[0].replace(" ", "").startswith("s3://")
+
+    if not use_torchdata:
+        urls = urls[0]
```

Comment:
I don't understand this. use_torchdata is equivalent to shards_on_s3. What if someone passes in something like ["/path/to/shard1", "path/to/shard2"]. This code would make urls  "/path/to/shard1"